lerobot/common/robot_devices/motors/dynamixel.py

```diff
@@ -268,52 +353,279 @@ def connect(self):
             )
             raise
 
-        self.port_handler.setBaudRate(BAUD_RATE)
+        # Allow to read and write
+        self.is_connected = True
+
         self.port_handler.setPacketTimeoutMillis(TIMEOUT_MS)
 
+        # Set expected baudrate for the bus
+        self.set_bus_baudrate(BAUDRATE)
+
+        if not self.are_motors_configured():
+            print(
+                r"/!\ First, verify that all the cables are connected the proper way. If you detect an issue, before making any modification, unplug the power cord to not damage the motors. Rewire correctly. Then plug the power again and relaunch the script."
+            )
+            print(
+                r"/!\ Secondly, if the cables connection look correct and it is the first time that you use these motors, follow these manual steps to configure them."
+            )
+            input("Press Enter to configure your motors...")
+            print()
+            self.configure_motors()
+
+    def reconnect(self):
+        self.port_handler = PortHandler(self.port)
+        self.packet_handler = PacketHandler(PROTOCOL_VERSION)
+        if not self.port_handler.openPort():
+            raise OSError(f"Failed to open port '{self.port}'.")
         self.is_connected = True
 
+    def are_motors_configured(self):
+        try:
+            return (self.motor_indices == self.read("ID")).all()
```

Comment:
Added comment

```python
    def are_motors_configured(self):
        # Only check the motor indices and not baudrate, since if the motor baudrates are incorrect,
        # a ConnectionError will be raised anyway.
        try:
            return (self.motor_indices == self.read("ID")).all()
        except ConnectionError as e:
            print(e)
            return False
            ```